identity-iota/Cargo.toml

```diff
@@ -19,6 +19,7 @@ futures = { version = "0.3" }
 identity-core = { version = "=0.5.0-dev.4", path = "../identity-core" }
 identity-credential = { version = "=0.5.0-dev.4", path = "../identity-credential" }
 identity-did = { version = "=0.5.0-dev.4", path = "../identity-did" }
+identity-iota-core = { version = "=0.5.0-dev.4", path = "../identity-iota-core" }
```

Comment:
We should really be passing `default-features = false` to our own crates too and re-exposing feature flags or enabling them as-needed.